pkg/sm/plugins/ufm/ufm.go

```diff
@@ -110,21 +108,18 @@ func (u *ufmPlugin) AddGuidsToPKey(pKey int, guids []net.HardwareAddr) error {
 		pKey, strings.Join(guidsString, ",")))
 
 	if _, err := u.client.Post(u.buildUrl("/ufmRest/resources/pkeys"), http.StatusOK, data); err != nil {
-		err = fmt.Errorf("AddGuidsToPKey(): failed to add guids %v to PKey 0x%04X "+
-			"with error: %v", guids, pKey, err)
-		glog.Error(err)
+		err = fmt.Errorf("failed to add guids %v to PKey 0x%04X with error: %v", guids, pKey, err)
 		return err
 	}
 
 	return nil
 }
 
 func (u *ufmPlugin) RemoveGuidsFromPKey(pKey int, guids []net.HardwareAddr) error {
-	glog.V(3).Infof("RemoveGuidsFromPKey(): pkey 0x%04X, guids %v", pKey, guids)
+	log.Debug().Msgf("RemoveGuidsFromPKey(): pkey 0x%04X, guids %v", pKey, guids)
 
 	if !ibUtils.IsPKeyValid(pKey) {
-		err := fmt.Errorf("RemoveGuidsFromPKey(): Invalid pkey 0x%04X, out of range 0x0001 - 0xFFFE", pKey)
-		glog.Error(err)
+		err := fmt.Errorf("invalid pkey 0x%04X, out of range 0x0001 - 0xFFFE", pKey)
```

Comment:
just return instead of assigen to err. It is many places not just this

Reply:
@moshe010 fixed in all locations